src/scene/renderer/forward-renderer.js

```diff
@@ -930,12 +934,12 @@ class ForwardRenderer {
             const light = lights[i];
 
             if (light.enabled) {
-
                 // directional lights are marked visible at the start of the frame
                 if (light._type !== LIGHTTYPE_DIRECTIONAL) {
                     light.getBoundingSphere(tempSphere);
                     if (camera.frustum.containsSphere(tempSphere)) {
                         light.visibleThisFrame = true;
+                        light.usePhysicalUnits = this.scene.physicalUnits;
```

Comment:
add `const scenePhysicalUnits = this.scene.physicalUnits;` outside of the loop, and use it inside the loop to avoid class member lookups.